pype/modules/ftrack/__init__.py

```diff
@@ -1,7 +1,14 @@
+import os
+
 from . import ftrack_server
 from .ftrack_server import FtrackServer, check_ftrack_url
 from .lib import BaseHandler, BaseEvent, BaseAction, ServerAction
 
+from pype.api import system_settings
+
+os.environ["FTRACK_SERVER"] = (
```

Comment:
I would just add `# TODO set in ftrack module` to keep record that this is not place where this should happen